UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/MacroTransformer.cs

```diff
@@ -27,7 +27,7 @@ string now_Year(Match match)
 
             string nowAddDays_Year(Match match)
             {
-                var dayOffset = offsetCapture(match);
+                int dayOffset = offsetCapture(match);
```

Comment:
Can we declare it at class level e.g. private int dayOffset  = 0; and use it ?
```suggestion
                dayOffset = offsetCapture(match);
```

Reply:
Resolved